libs/server-sdk/include/launchdarkly/server_side/data_source_status.hpp

```diff
@@ -61,6 +61,12 @@ enum class ServerDataSourceState {
      * explicitly closed.
      */
     kOff = 3,
+
+    /**
+     * Indicates that the application has told the SDK to stay offline.
+     */
+    kSetOffline = 4,
```

Comment:
Wasn't sure if `kOff` was appropriate/intended for offline mode.

Reply:
`kSetOffline` is a client side data source state, but doesn't seem to be a server side one. I think it would be kOff. 

(Go and dotnet only have the states I already defined here, but I don't really see an actual spec.) 

Neither of them say, in the documentation, which it is.

That said, even for client, it shouldn't be set offline. I think SetOffline is only that you dynamically made it offline. Where offline in the config is shutdown.